src/backend/tests/reimbursement-requests.test.ts

```diff
@@ -222,4 +222,49 @@ describe('Reimbursement Requests', () => {
       expect(prisma.reimbursement_Request.update).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('Delivered Tests', () => {
+    test('Mark as delivered fails for non submitter', async () => {
+      jest
+        .spyOn(prisma.reimbursement_Request, 'findUnique')
+        .mockResolvedValue({ ...GiveMeMyMoney, dateDelivered: new Date('12/25/203') });
+
+      await expect(
+        ReimbursementRequestService.markReimbursementRequestAsDelivered(wonderwoman, GiveMeMyMoney.reimbursementRequestId)
+      ).rejects.toThrow(new AccessDeniedException('Only the creator of the reimbursement request can mark as delivered'));
+    });
+
+    test('Mark as delivered fails for undefined ID', async () => {
+      jest
+        .spyOn(prisma.reimbursement_Request, 'findUnique')
+        .mockResolvedValue({ ...GiveMeMyMoney, dateDelivered: new Date('12/25/203') });
```

Comment:
you want prisma to return null when the id doesnt relate to anything so mock resolve this as null